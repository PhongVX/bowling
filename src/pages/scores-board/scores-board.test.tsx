import React from "react";
import { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import { ScoresBoardPage } from ".";

describe("GetStartedPage Snapshot", () => {
  it("renders correctly and matches snapshot", () => {
    const { container } = render(
      <BrowserRouter>
        <ScoresBoardPage />
      </BrowserRouter>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
