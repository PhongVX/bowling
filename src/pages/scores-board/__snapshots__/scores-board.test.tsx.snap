// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GetStartedPage Snapshot renders correctly and matches snapshot 1`] = `
<div
  class="scores-board-wrapper"
>
  <div>
    <h1
      class="heading scores-board-heading"
      test-id="scores-board-heading"
    >
      Bowling Scores Board
    </h1>
  </div>
  <div
    style="justify-content: center; display: flex;"
  >
    <table>
      <tr
        class="scores-board-header"
      >
        <th
          class="scores-board-header"
        >
          Player
        </th>
        <th
          class="scores-board-header"
        >
          1
        </th>
        <th
          class="scores-board-header"
        >
          2
        </th>
        <th
          class="scores-board-header"
        >
          3
        </th>
        <th
          class="scores-board-header"
        >
          4
        </th>
        <th
          class="scores-board-header"
        >
          5
        </th>
        <th
          class="scores-board-header"
        >
          6
        </th>
        <th
          class="scores-board-header"
        >
          7
        </th>
        <th
          class="scores-board-header"
        >
          8
        </th>
        <th
          class="scores-board-header"
        >
          9
        </th>
        <th
          class="scores-board-header"
        >
          10
        </th>
        <th
          class="scores-board-header"
        >
          Total
        </th>
      </tr>
    </table>
  </div>
</div>
`;
